datajoint/declare.py

```diff
@@ -196,6 +197,10 @@ def declare(full_table_name, definition, context):
     :param definition: DataJoint table definition
     :param context: dictionary of objects that might be referred to in the table.
     """
+
+    table_name = full_table_name.strip('`').split('.')[1]
+    if len(table_name) > MAX_TABLE_NAME_LENGTH:
+        raise DataJointError('Table name `%s` is too long')
```

Comment:
I think it'll be helpful to show what is the maximum table length along with this error message.

Reply:
fixed